src/main/java/core/Car.java

```diff
@@ -0,0 +1,30 @@
+package core;
+
+
+public class Car implements CarAction{
+    static final int GOING_NUM = 4;
+    public int position;
+
+    public Car(int position) {
+        this.position = position;
+    }
+
+    public void carAction(int randNum) {
+        move(isMove(randNum));
+    }
```

Comment:
```suggestion
    public void carAction(CarAction carAction) {
        move(carAction.isMove());
    }
```
이런식으로 할 경우 car의 move 조건을 외부에서 주입할 수 있기 때문에 조금 더 유연하게 대처할 수 있습니다.